src/features/click-to-play.js

```diff
@@ -1411,17 +1403,25 @@ export function init (args) {
     })
 
     // Request the current state of Click to Load from the platform.
-    // Note: When the response is received, initCTL() is then called by the
-    //       response handler to finish starting up the feature.
+    // Note: When the response is received, the response handler finishes
+    //       starting up the feature.
     sendMessage('getClickToLoadState')
 }
 
 export function update (args) {
-    const detail = args && args.detail
-    if (!(detail && detail.func)) { return }
+    // Unsupported update message.
+    if (!args?.detail) return
 
-    const fn = updateHandlers[detail.func]
-    if (typeof fn !== 'function') { return }
+    // Message response.
+    if (typeof args?.detail?.func === 'string' && knownMessage(args.detail.func)) {
+        return messageResponseHandlers[args.detail.func](args.detail.response)
+    }
 
-    fn(detail.response)
+    // Other known update messages.
+    if (args?.detail?.type === 'displayClickToLoadPlaceholders') {
+        // TODO: Pass `args.detail.ruleAction` through, that way only content
```

Comment:
I actually think this is OK for now, but I thought it made sense to pass the rule action through ready for if we ever decide to do this. Otherwise, changing the message on all platforms would be a pain.